### PR TITLE
Add octavius.1.2.1

### PR DESCRIPTION
**Pull request by lpw25**
Bugfix release so that it works with 4.08

### CODE REVIEW

File: packages/octavius/octavius.1.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+homepage: "https://github.com/ocaml-doc/octavius"
+doc: "http://ocaml-doc.github.io/octavius/"
+license: "ISC"
+dev-repo: "git+http://github.com/ocaml-doc/octavius.git"
+bug-reports: "https://github.com/ocaml-doc/octavius/issues"
+tags: ["doc" "ocamldoc" "org:ocaml-doc"]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "ocamlfind" {build}
```
Comment (kit-ty-kate) [inline packages/octavius/octavius.1.2.1/opam]:
```suggestion
```
`octavius` doesn't seem to use ocamlfind at all in its source code

Comment (lpw25) [reply packages/octavius/octavius.1.2.1/opam]: It uses ocamlbuild with `-use-ocamlfind`

Comment (lpw25) [reply packages/octavius/octavius.1.2.1/opam]: Wait, no it doesn't do that anymore, I forgot that it got changed to jbuilder

**Comment by mseri** [comment]
Thanks

**Pull request merged**